src/combinatorics/set_family/kleitman.lean

```diff
@@ -0,0 +1,83 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import combinatorics.set_family.harris_kleitman
+import combinatorics.set_family.intersecting
+
+/-!
+# Kleitman's bound on the size of intersecting families
+
+An intersecting family on `n` elements has size at most `2ⁿ⁻¹`, so we could naïvely think that two
+intersecting families could cover all `2ⁿ` sets. But actually that's not case because for example
+none of them can contain the empty set. Intersecting families are in some sense correlated.
+Kleitman's bound stipulates that `k` intersecting families cover at most `2ⁿ - 2ⁿ⁻ᵏ` sets.
+
+Note that, as opposed to the one family case, not all maximal unions of `k` families have this size.
```

Comment:
Since this part of the result isn't in the file yet, this line in the module doc is more confusing than helpful.